web/public/locales/en/views/system.json

```diff
@@ -153,11 +153,17 @@
     "title": "Enrichments",
     "infPerSecond": "Inferences Per Second",
     "embeddings": {
+      "image_embedding": "Image Embedding",
+      "text_embedding": "Text Embedding",
+      "face_recognition": "Face Recognition",
+      "plate_recognition": "Plate Recognition",
       "image_embedding_speed": "Image Embedding Speed",
       "face_embedding_speed": "Face Embedding Speed",
       "face_recognition_speed": "Face Recognition Speed",
       "plate_recognition_speed": "Plate Recognition Speed",
-      "text_embedding_speed": "Text Embedding Speed"
+      "text_embedding_speed": "Text Embedding Speed",
```

Comment:
Are these actually used?

Reply:
Yep

Reply:
Can you point me to where? I don't see changes on the enrichments metrics

Reply:
https://github.com/blakeblackshear/frigate/blob/f9b2db440542cf24d636f549478336c150ddbe05/web/src/views/system/EnrichmentMetrics.tsx#L81

Reply:
Introduced in https://github.com/blakeblackshear/frigate/pull/17682